examples/simple.rs

```diff
@@ -75,19 +74,13 @@ fn send_packets(mut net: ResMut<NetworkResource>, time: Res<Time>, args: Res<Arg
         }
     }
 }
-
-#[derive(Default)]
-struct NetworkReader {
-    network_events: EventReader<NetworkEvent>,
-}
-
 fn handle_packets(
     mut net: ResMut<NetworkResource>,
     time: Res<Time>,
-    mut state: ResMut<NetworkReader>,
     network_events: Res<Events<NetworkEvent>>,
 ) {
-    for event in state.network_events.iter(&network_events) {
+    let mut reader = network_events.get_reader();
+    for event in reader.iter(&network_events) {
```

Comment:
```suggestion
    for event in network_events.get_reader().iter(&network_events) {
```

Reply:
Looks like `.get_reader()` is not needed. https://github.com/bevyengine/bevy/blob/v0.5.0/examples/ecs/event.rs#L46

Reply:
`get_reader()` is needed. There's many ways to iterate through events, we can use a `EventReader<Event>` from Resource, or use the `get_reader()` from events.